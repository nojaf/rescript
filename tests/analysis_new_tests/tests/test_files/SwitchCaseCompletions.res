// == TEST: Empty case, array
let someStringArr = ["hello"]

let x = switch someStringArr {
  | 
//  ^com
}

// == TEST: Empty case, record
let x = switch TestTypeDefs.nestedTestRecord {
  | 
//  ^com
}

// == TEST: Empty case, bool
let x = switch true {
  | 
//  ^com
}
