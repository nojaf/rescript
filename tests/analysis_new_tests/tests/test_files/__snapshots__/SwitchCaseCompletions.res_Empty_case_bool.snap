Found Completable: Cpattern

1 + // Empty case, bool
2 + let x = switch true {
                   ----  
3 +   | 
4 + //  ^com
5 + }

[{
    "label": "true",
    "kind": 12,
    "tags": [],
    "detail": "bool",
    "documentation": null,
    "sortText": "A",
    "insertText": "true",
    "insertTextFormat": 2
  }, {
    "label": "false",
    "kind": 12,
    "tags": [],
    "detail": "bool",
    "documentation": null,
    "sortText": "A",
    "insertText": "false",
    "insertTextFormat": 2
  }]

